src/main/kotlin/apply/application/MissionDtos.kt

```diff
@@ -126,6 +124,28 @@ data class MyMissionResponse(
     )
 }
 
+data class MyMissionResponse(
+    val id: Long,
+    val title: String,
+    val description: String,
+    val submittable: Boolean,
+    val startDateTime: LocalDateTime,
+    val endDateTime: LocalDateTime,
+    val status: MissionStatus,
+    val submitted: Boolean,
+) {
+    constructor(mission: Mission, submitted: Boolean, formattedDescription: String) : this(
```

Comment:
a: 사소하지만 부 생성자의 매개 변수 순서를 주 생성자와 동일하게 설정하면 어떨까요?

```suggestion
    constructor(mission: Mission, description: String, submitted: Boolean) : this(
```